StyleCop.Analyzers/StyleCop.Analyzers.Test/Properties/AssemblyInfo.cs

```diff
@@ -33,3 +34,6 @@
 [assembly: AssemblyVersion("1.0.0.0")]
 [assembly: AssemblyFileVersion("1.0.0.0")]
 [assembly: AssemblyInformationalVersion("1.0.0.0-dev")]
+
+// XUnit
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerClass, DisableTestParallelization = true, MaxParallelThreads = 8)]
```

Comment:
:exclamation: This is the cause of the tests not completing. Revert the changes to this file for now and we can consider adding it back later.